spec/system/admin/instrument_schedule_positions_spec.rb

```diff
@@ -25,6 +25,8 @@
   describe "as a director" do
     let(:user) { create(:user, :facility_director, facility: facility) }
 
+    # Some of these specs have been failing even though the arrays look the same in the error message.
```

Comment:
Should this be a `TODO:` comment?
Where is the `appear_in_order` matcher defined?

Reply:
https://github.com/tablexi/nucore-open/blob/master/spec/support/matchers/display_order_matchers.rb